package.json

```diff
@@ -117,6 +120,7 @@
     "react-day-picker": "6.0.2",
     "react-docgen": "2.13.0",
     "react-dom": "15.4.0",
+    "react-hot-loader": "1.3.1",
```

Comment:
Docker builds are faster because they use `npm install --production`, and that installs less dependencies. That's enough benefit in my book :)